Dnn.AdminExperience/Library/Dnn.PersonaBar.UI/admin/personaBar/scripts/main.js

```diff
@@ -770,7 +770,7 @@ require(['jquery', 'knockout', 'moment', '../util', '../sf', '../config', './../
                         };
 
                         viewModel.updateText = ko.computed(function() {
-                            return viewModel.updateType() === 2 ? util.resx.PersonaBar.CriticalUpdate : util.resx.PersonaBar.NormalUpdate;

```

Comment:
@mitchelsellers we used to have "Update" and "Critical Update", is that scenario still a supported use case ?

Reply:
The code still exists, but in our current procedure I can't see (1) how we'd do it in Github and (2) if it is really useful/accurate/effective. Most of our upgrades are critical. But still few people upgrade ...

Reply:
Wouldn't normal vs. critical be based on security bulletins? We already have code in the Security extension of the Persona Bar to read that data. https://github.com/dnnsoftware/Dnn.Platform/blob/3e0d9c7c12e08f62ab458304aecdbc42b540d893/Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/SecurityController.cs#L608-L685

Reply:
yeah, that's what I was talking about just did not know where it came from in code...